apps/emqx_mq/src/emqx_mq.app.src

```diff
@@ -0,0 +1,10 @@
+{application, emqx_mq, [
+    {description, "Message Queue"},
+    {vsn, "0.1.0"},
+    {registered, []},
+    {mod, {emqx_mq_app, []}},
+    {applications, [kernel, stdlib, gproc, emqx, emqx_durable_storage]},
```

Comment:
Business applications should have `emqx_ds_backends` as a dependency rather then `durable_storage`.